src/staticanalysis/bot/static_analysis_bot/infer/__init__.py

```diff
@@ -27,6 +27,8 @@
 ac_add_options --with-android-sdk="{mozbuild}/android-sdk-linux/android-sdk-linux"
 ac_add_options --with-android-ndk="{mozbuild}/android-ndk/android-ndk"'''
 
+RES_DIRECTORY = '/tmp/mozilla-state'
```

Comment:
You should use `os.environ['MOZBUILD_STATE_PATH']` as it's [defined here](https://github.com/mozilla/release-services/blob/master/src/staticanalysis/bot/default.nix#L177)